framework/components/ecl-forms/ecl-forms-legends/README.md

```diff
@@ -1 +1,4 @@
-# Legends
```

Comment:
Well let's say like every other component, except button :)
We will have to synchronize here